edn/src/types.rs

```diff
@@ -248,7 +389,32 @@ impl Ord for Value {
     }
 }
 
+impl PartialOrd for SpannedValue {
+    fn partial_cmp(&self, other: &SpannedValue) -> Option<Ordering> {
+        Some(self.cmp(other))
+    }
+}
+
+impl Ord for SpannedValue {
+    fn cmp(&self, other: &SpannedValue) -> Ordering {
+        Value::from(self.clone()).cmp(&Value::from(other.clone()))
```

Comment:
As discussed on Slack, cloning isn't necessary, just a shortcut I took in order to get something that is landable (as in: provides value that doesn't exist yet, is better than nothing etc.). Changing this implementation to avoid cloning is relatively trivial, sufficient for a good followup good first bug, but I'll do it in this PR.